backend/prisma/schema.prisma

```diff
@@ -93,3 +99,41 @@ model ValBug {
   text String
   patchnote Patchnotes_val @relation(fields: [patchId], references: [id])
 }
+
+model Comment_ow {
+  id Int @id @default(autoincrement())
+  message String
+  voteCount Int @default(0)
+  votes_ow Vote_ow[]
+  patchnotes_ow Patchnotes_ow @relation(fields:[patchId], references:[id])
+  patchId Int
+  user User @relation(fields:[userId], references:[id])
+  userId Int
+}
+
+model Comment_val {
+  id Int @id @default(autoincrement())
+  message String
+  voteCount Int @default(0)
+  votes_val Vote_val[]
+  patchnotes_val Patchnotes_val @relation(fields:[patchId], references:[id])
+  patchId Int
+  user User @relation(fields:[userId], references:[id])
+  userId Int
+}
+
+model Vote_ow {
+  id Int @id @default(autoincrement())
+  user User @relation(fields:[userId], references:[id])
+  userId Int
+  comment Comment_ow @relation(fields:[commentId], references:[id])
+  commentId Int
+}
+
+model Vote_val {
+  id Int @id @default(autoincrement())
+  user User @relation(fields:[userId], references:[id])
+  userId Int
+  comment Comment_val @relation(fields:[commentId], references:[id])
+  commentId Int
+}
```

Comment:
we shouldn't be splitting votes and comments by game like this. Notice how all of the fields are identical except the ones that are differentiated by game. Instead you should just have a field with a `game` string or id.

Reply:
Yep, I will be working on created a unified model for comments, votes, replies, etc